app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It is a good practice to use type hints for all function parameters and return types. For the `friends` parameter, you could specify that it's a list of a specific type, if applicable. For example, if `friends` is a list of `Friend` objects, you could annotate it like this: `friends: List[Friend]`.